network_modeling/models/cray-network-architecture.yaml

```diff
@@ -64,6 +68,11 @@ network_v2:
       connections:
         - name: "river_ncn_leaf"
           speed: 25
+    - name: "slingshot_hsn_switch"
+      model: "sw-hsn"
```

Comment:
The model shouldn't be sw-hsn, but the same as you have for name "slingshot_hsn_switch".  This will need to match up elsewhere as this is the "primary key" that links model, lookup and hardware.   Reasoning is that the model is either a "real model" - "3600M_ABC" - or a human readable descriptive thing like "river_ncn_2_node".  My convention is underscore, not dashes.